search-app/src/cmr/search/data/elastic_relevancy_scoring.clj

```diff
@@ -58,17 +58,27 @@
 
    The algorithm is to currently compare scores in the following order (we only go to the next level
    in the scoring system in the case of a tie at the higher level):
-   1. keyword boost
-   2. temporal overlap
-   3. community usage
-   4. temporal end date of the collection
-   5. processing level."
+   1. community usage, if usage_score sort key present
+   2. keyword boost
+   3. temporal overlap
+   4. relevancy with community usage secondary
```

Comment:
What does it mean exactly?  could you make it more clearly like "community usage, if ....  "  just like #1 above?
Also, could you correct the doc string of sort-use-relevancy-score in your ticket? there seem to be some typos there  which makes it difficult to read...